app/controllers/spam_controller.rb

```diff
@@ -1,20 +1,28 @@
 class SpamController < ApplicationController
   module ModerationGuards
     def spam_check(map)
-      #check and spam only unspammed maps
+      # check and spam only unspammed maps
       map.spam unless map.status == Map::Status::BANNED
     end
   
     def ban_check(map)
-      #check and ban only unbanned non-anonymous authors
+      # check and ban only unbanned non-anonymous authors
       map.user.ban unless map.anonymous? || map.user.status == User::Status::BANNED
     end
+
+    def publish_check(map)
```

Comment:
Do you think it's worth using a longer method name that reads more like human speech, like `check_and_publish`? Just an idea. The comments make it clear but i'm thinking when it's used in other files.

Reply:
Yeah, it's definitely worth it. I'm all for longer method-names for better readability. Will use your suggestions and re-push. Thank you ❤️